main.py

```diff
@@ -257,11 +259,18 @@ async def main(args: dict, cpid_to_market_info: dict[int, dict]):
                 *tasks
             )
     elif conf['use_websocket']:
-        joined = ",".join([str(x) for x in cpids])
+        params = []
+        if cpids:
+            joined_cpids = ",".join([str(x) for x in cpids])
+            params.append(f"clobPairIds={joined_cpids}")
+
+        if subaccount_ids:
+            joined_subaccount_ids = ",".join([str(x) for x in subaccount_ids])
+            params.append(f"subaccountIds={joined_subaccount_ids}")
         websocket_port = conf['dydx_full_node']['websocket_port']
-        websocket_addr = f"ws://{host}:{websocket_port}/ws?clobPairIds={joined}"
+        params_str = "&".join(params)
+        websocket_addr = f"ws://{host}:{websocket_port}/ws?{params_str}"
```

Comment:
ah i see you a added the subaccount ids, feel free to close my pr

https://github.com/dydxprotocol/grpc-stream-client/pull/12/

But could you add the change for the config yaml 9092 port in here?